client/src/app/(modules)/network/(main)/(details)/field.tsx

```diff
@@ -74,13 +96,31 @@ const Field = ({ label, value, url, external, type, hasEllipsis }: Field & { typ
         renderLink(url, external)
       ) : (
         <div>
-          <div
-            ref={ref}
-            className={cn('text-sm', {
-              'line-clamp-2': !isExpanded && isOverTwoLines,
-            })}
-          >
-            {value}
+          <div className="flex items-center space-x-2">
+            <div
+              ref={ref}
+              className={cn('text-sm', {
+                'line-clamp-2': !isExpanded && isOverTwoLines,
+              })}
+            >
+              {value}
+            </div>
+            {description && (
+              <TooltipProvider>
+                <Tooltip delayDuration={0} open={openInfo} onOpenChange={setInfoOpen}>
+                  <TooltipTrigger asChild onClick={handleInfoClick}>
+                    <Button type="button" size="auto" variant="icon">
+                      <span className="sr-only">Project type description</span>
```

Comment:
We've renamed “projects” as “initiatives” already, so you would have to change this text. But I was thinking that if we want to make this more scalable, maybe we should just write “Description”. What do you think?